src/frontend/web_application/src/layouts/Page/style.scss

```diff
@@ -0,0 +1,20 @@
+// --- Base ---
+// global rules
+$co-font-path: '../../fonts';
+@import '../../styles/base/fonts';
+@import '../../styles/base/reset';
+
+// --- Vendor ---
+@import '../../styles/vendor/bootstrap_foundation-sites';
+
+@import '../../styles/common';
+
+.l-body {
+  &__content {
+    // XXX: need a better fix for call-to-action override on mobile
+    margin-bottom: 160px;
+    padding-top: 45px;
+
+    background-color: $co-color__fg__back;
```

Comment:
I prefer not here. We need the background to fit all use cases (Settings, Discussions, Contacts etc.)